scripts/test_env.sh

```diff
@@ -118,8 +183,22 @@ function clean_up_peers {
 }
 
 case $1 in
-
     setup)
+        declare -i N_PEERS
+        if [ -z "$3" ]
+        then 
+            echo "Number of peers is not provided, using default value of $DEFAULT_N_PEERS"
+            N_PEERS="$DEFAULT_N_PEERS"
```

Comment:
See comment on line 10. Silent fallback might not be the most intuitive, but it is what we do for a few other variables.